server/repository/src/db_diesel/stock_line.rs

```diff
@@ -170,7 +173,10 @@ type BoxedStockLineQuery = IntoBoxed<
     DBType,
 >;
 
-fn create_filtered_query(filter: Option<StockLineFilter>) -> BoxedStockLineQuery {
+fn create_filtered_query(
+    filter: Option<StockLineFilter>,
+    store: Option<String>,
```

Comment:
store_id please

Reply:
store_id is already being used as a filter >.>